### PR TITLE
Task-50-DB-Structure

### PR DESCRIPTION
**Pull request by OlegPanasyuk**
-	Adding DB Structure version 2

-	Creating wrapper for versions of DB

-	Separating versions of DB in Project with two items v1, v2

### CODE REVIEW

File: database/v2/schemas/public_profile.model.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        type: type.TEXT,
+        allowNull: true,
+      },
+      linkedInLink: {
+        type: type.TEXT,
+        allowNull: true,
+      },
+      faceBookLink: {
+        type: type.TEXT,
+        allowNull: true,
```
Comment (slavasgod) [inline database/v2/schemas/public_profile.model.js:20]: Replace `faceBookLink` to `facebookLink`. Yes, looks like a joke, but this is an official name)

File: database/v2/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    host: process.env.DB_HOST,
+    dialect: 'postgres',
+    dialectOptions: {
+      ssl: true,
+    },
+    logging: true,
+  }
+);
+
+Object.keys(models).forEach(f => {
```
Comment (melonholy) [inline database/v2/index.js]: What does 'f' mean?

**Review by slavasgod — CHANGES_REQUESTED** [review]

**Review by melonholy — CHANGES_REQUESTED** [review]

File: database/v2/schemas/public_profile.model.js (hunk 1 above)
Comment (logoped583st) [reply database/v2/schemas/public_profile.model.js:20]: i don't think link is good naming for identifier, also i think we can avoid using Text type here. Make investigate about size of facebook,linkedin token

File: database/v2/schemas/setts_profile.model.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+module.exports = (sequelize, type) => {
+  const MODEL = sequelize.define(
+    'SettsProfiles',
```
Comment (logoped583st) [inline database/v2/schemas/setts_profile.model.js:3]: SettsProfiles ?

File: database/v2/index.js (hunk 2 above)
Comment (OlegPanasyuk) [reply database/v2/index.js]: Ok, I will change this short name with a long readable one.

File: database/v2/schemas/setts_profile.model.js (hunk 3 above)
Comment (OlegPanasyuk) [reply database/v2/schemas/setts_profile.model.js:3]: > SettsProfiles ?

Can you write more spread comments?

File: database/v2/schemas/public_profile.model.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+module.exports = (sequelize, type) => {
+  const MODEL = sequelize.define(
+    'PublicProfiles',
+    {
+      id: {
+        type: type.INTEGER,
```
Comment (logoped583st) [inline database/v2/schemas/public_profile.model.js:6]: id type for PublicProfiles is Integer, but this id has a relation with Users  where id is UUID.

File: database/v2/schemas/public_profile.model.js (hunk 1 above)
Comment (OlegPanasyuk) [reply database/v2/schemas/public_profile.model.js:20]: Ok. I'll search for such information/

File: database/v2/schemas/users.model.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    'Users',
+    {
+      email: {
+        type: type.STRING(64),
+        allowNull: false,
+      },
+      password: {
+        type: type.STRING(100),
+      },
+      id: {
```
Comment (logoped583st) [inline database/v2/schemas/users.model.js:12]: what sense of using uuid as id for users and other entities ?

File: database/v2/schemas/public_profile.model.js (hunk 1 above)
Comment (OlegPanasyuk) [reply database/v2/schemas/public_profile.model.js:20]: So those fields are for storing URL to their accounts.
We decided to not store token

File: database/v1/index.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+Object.keys(db).forEach(modelName => {
+  if (db[modelName].associate) {
+    db[modelName].associate(db);
+  }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
```
Comment (logoped583st) [inline database/v1/index.js:33]: Does this line make sense?

Comment (OlegPanasyuk) [reply database/v1/index.js:33]: It has a sense.  
`sequelize `is an instance of Class which contents and maintenance connection to db.
`Sequelize` is Class which contents types and operators.

File: database/v2/schemas/public_profile.model.js (hunk 4 above)
Comment (OlegPanasyuk) [reply database/v2/schemas/public_profile.model.js:6]: Fixed

File: database/v2/schemas/users.model.js (hunk 5 above)
Comment (OlegPanasyuk) [reply database/v2/schemas/users.model.js:12]: Fixed

**Review by melonholy — APPROVED** [review]

**Review by slavasgod — APPROVED** [review]

**Pull request merged**